pyLucXor/peptide.py

```
@@ -0,0 +1,907 @@
+        Returns:
+            str: Unmodified peptide sequence
+        """
+        try:
+            # Remove all modification markers
+            unmod_seq = ""
+            i = 0
+            while i < len(self.peptide):
+                if self.peptide[i:i+9] == "(Phospho)":
+                    i += 9
+                elif self.peptide[i:i+9] == "(Oxidation)":
+                    i += 9
```

Ruff (0.12.2)

Combine `if` branches using logical `or` operator

Combine `if` branches

(SIM114)